doc/api/fs.md

```diff
@@ -3311,6 +3311,38 @@ a colon, Node.js will open a file system stream, as described by
 Functions based on `fs.open()` exhibit this behavior as well:
 `fs.writeFile()`, `fs.readFile()`, etc.
 
+### `fs.openAsBlob(path)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `path` {string|Buffer|URL}
+
+Returns a {Blob} whose data is backed by the given file.
```

Comment:
Perhaps as a separate PR. Using `File` right now would require an extra bit of work that I'd prefer to separate out into a separate PR. For example, `File` uses private fields instead of hidden symbols which makes it difficult to set the name and last modified properties from something other than the constructor. I'd prefer to just do that work in a separate PR given that this one is already quite large. This new API is being introduced as Experimental so we can make these changes separately before it goes stable.